.github/workflows/test.yml

```diff
@@ -27,6 +27,10 @@ jobs:
         with:
           go-version-file: 'go.mod'
           cache: true
+      - name: Run linters
```

Comment:
Nit: It may be worth running this after `go mod download` at least (I'd say even after `go build`) so that the module downloads or compilation errors stop the workflow instead of the linting action.

Reply:
sounds good, fixed. 🚀